.github/workflows/visual-diff.yml

```diff
@@ -12,8 +12,8 @@ on:
 
 jobs:
   visual-diff:
-    # This is the opposite condition as acceptance.yml/visual-diff
-    if: github.head_repository.full_name != 'getsentry/sentry'
+    # Only execute this check when a PR is opened from a fork rather than the upstream repo & master pushes
+    if: github.event.workflow_run.head_repository.full_name != 'getsentry/sentry' || github.event.workflow_run.head_branch == 'master'
```

Comment:
Ideally, `master` pushes would execute `visual-diff` from the acceptance workflow, however, as I found out in PR #33964 we can't at the moment.

This screenshot shows that a push to `master` would trigger the workflow run while a push to a PR would not:
<img width="511" alt="image" src="https://user-images.githubusercontent.com/44410/165358822-f3a4a90b-5c0a-4bcf-8e6c-7621b05337b8.png">

Reply:
This also shows that a check on a PR for a fork would execute (unlike the PR from within the upstream repo):
<img width="541" alt="image" src="https://user-images.githubusercontent.com/44410/165359328-a7f7f4dc-4363-4486-aa2d-3f997a836a4d.png">

Reply:
just checking, if a branch in a fork is named `master` does it still do the right thing?

Reply:
> just checking, if a branch in a fork is named `master` does it still do the right thing?

Thanks for checking. We have never supported that. We only use workflow_run for PRs from Sentry forks. That will help us make sure their changes do not regress the acceptance tests.